piplicenses.py

```diff
@@ -432,20 +445,16 @@ def find_license_from_classifier(message):
             if license != 'OSI Approved':
                 licenses.append(license)
 
-    if len(licenses) > 0:
-        license_from_classifier = ', '.join(licenses)
-
-    return license_from_classifier
+    return licenses
```

Comment:
Just my little opinion, the return value of the function `find_license_from_classifier` was changed from `str` to `list`, which made other implementation and test codes more complicated.

But it's not important enough to oppose the merge.

Reply:
You're right that it made the commit a bit more complicated. However I believe this is the better solution here. Changing the responsibility of this function to only extract the license classifier might make the work down the line a bit easier.